docs/security.md

```diff
@@ -0,0 +1,194 @@
+# Security
+
+The OpenTelemetry Collector defaults to operating in a secure manner, but is
+configuration driven. This document captures important security aspects and
+considerations for the Collector. This document is intended for both end-users
+and component developers. It assumes at least a basic understanding of the
+Collector architecture and functionality.
+
+> Note: Please review the [configuration
+> documentation](https://opentelemetry.io/docs/collector/configuration/)
+> prior to this security document.
+
+## TL;DR
+
+### End-users
+
+- Configuration
+  - SHOULD only enable the minimum required components
+  - SHOULD ensure sensitive configuration information is stored securely
+- Permissions
+  - SHOULD not run Collector as root/admin user
+  - MAY require privileged access for some components
+- Receivers/Exporters
+  - SHOULD use encryption and authentication
```

Comment:
I wrote a blog post some time ago on this, and it's available under the OpenTelemetry publication. Perhaps it could be linked here?

https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f

Reply:
Nice post! Absolutely.